playground/frontend/build.gradle

```diff
@@ -183,16 +183,30 @@ ext.deleteFilesByRegExp = { re ->
 }
 
 tasks.register("integrationTest") {
```

Comment:
Ran this command after following README instructions to download chromedriver.  Naturally as I was unable to run locally, the integration tests froze on the first URL.